packages/web/src/features/translation/TranslateWord.tsx

```diff
@@ -116,10 +130,10 @@ const TranslateWord = forwardRef<TranslateWordRef, TranslateWordProps>(
                       {capitalize(t('translate:saving'))}
                     </>
                   );
-                } else if (status === 'saved') {
+                } else if (status === 'approved') {
                   return (
                     <>
-                      <Icon icon="check" className="me-1" />
+                      <Icon icon="check" className="me-1 text-green-600" />
                       {capitalize(t('translate:saved'))}
```

Comment:
I forgot I need to update this string to say approved. That shouldn't take to long